diff --git a/dist/main.cjs b/dist/main.cjs
index d3060bf6cd530e6a68437dbe3eb00c2437b08af6..a6811a2c25b2247823735aa59998ad506e2562df 100644
--- a/dist/main.cjs
+++ b/dist/main.cjs
@@ -1,4 +1,4 @@
-"use strict";Object.defineProperties(exports,{__esModule:{value:!0},[Symbol.toStringTag]:{value:"Module"}});const t=require("fs"),e=require("path"),n=t=>t&&"object"==typeof t&&"default"in t?t:{default:t},r=n(t),i=n(e);
+"use strict";Object.defineProperties(exports,{__esModule:{value:!0},[Symbol.toStringTag]:{value:"Module"}});const t=/*fs*/({}),e=/*path*/({}),n=t=>t&&"object"==typeof t&&"default"in t?t:{default:t},r=n(t),i=n(e);
 /*! *****************************************************************************
 Copyright (c) Microsoft Corporation.
 
diff --git a/package.json b/package.json
index 37df6ad3d326edbd37f2cb30a1491637fbf30673..d21568a6f4fa8f3bad09f77ba964a9e60835bceb 100644
--- a/package.json
+++ b/package.json
@@ -6,6 +6,7 @@
   "main": "./dist/main.cjs",
   "module": "./dist/main.js",
   "exports": {
+    "browser": "./dist/main.cjs",
     "require": "./dist/main.cjs",
     "script": "./dist/index.js",
     "default": "./dist/main.js"
